dmn-engine/dmn-engine-drg/pom.xml

```diff
@@ -11,6 +11,9 @@
 
   <properties>
     <version.camunda>7.17.0</version.camunda>
+    <version.slf4j>1.7.26</version.slf4j>
+    <version.junit>4.13.1</version.junit>
+    <version.assertj>2.9.1</version.assertj>
```

Comment:
Should we update to something more recent, like 3.21.0 (that comes with Spring Boot 2.6.x for example)